app/components/full_width_action_component/full_width_action_component.html.erb

```diff
@@ -1,21 +1,21 @@
 <section class="full_width_action <%= @color_class[@color] %>">
-  <h3 class="full_width_action__title">
-    <%= @title %>
-  </h3>
-  <p class="full_width_action__content">
-    <%= content %>
-  </p>
-
-  <div class="full_width_action__btn">
-    <% if @color == "cream" %>
-      <%= render(LinkBtnLrgGreenComponent.new( link_url: @link_url)) do %>
-        <%= @link_text %>
+  <%= render(MaxWidthComponent.new()) do %>
```

Comment:
I think having this as a component is really handy for instances like this but I'm wondering if there are also occasions where we might want to just use the class instead, for example when you want to apply the max-width to the parent container & don't want to have to add another div for that - could we maybe store the css in a shared file instead of at component level?

Reply:
I guess we're getting into the realm of global utility classes instead of BEM then? Maybe that's fine.

I think I need a more concrete example about the parent container thing your describing because I'm not picturing it. I was sort of thinking we'd have a go to thing that performs one way and if we needed to step out of that we'd use our components own styles.

Reply:
I'm thinking about situations like the block quote I guess, where you might just want to apply the max-width to the <p> tag instead of needing to wrap it in an extra <div> with a new component. Is it bad design wise to make use of global utilities in addition to BEM? I feel like it would be neater here but I can also see why you'd be hesitant to do it

Reply:
I think I'm going to merge because you've not blocked it but I take your point that this component is essentially a very restrictive way of applying a global utility class.

I'm open to it being refactored out / modified in some way if it proves to not be useful in the future.